plugins/woocommerce/includes/emails/class-wc-email-customer-pos-refunded-order.php

```diff
@@ -476,6 +530,26 @@ public function order_item_totals( $total_rows, $order, $tax_display ) {
 
 			return $total_rows;
 		}
+
+		/**
+		 * Get the store name from POS settings.
+		 *
+		 * @return string
+		 */
+		private function get_pos_store_name() {
+			return $this->format_string(
+				get_option( 'woocommerce_pos_store_name', PointOfSaleDefaultSettings::get_default_store_name() )
+			);
+		}
```

Comment:
🗒️ This function is the same as added in https://github.com/woocommerce/woocommerce/pull/58012. I'll watch out for potential merge conflicts when handling these two PRs.